content/courses/program-optimization/lookup-tables.md

```diff
@@ -376,137 +360,149 @@ const transactionSignature = await connection.sendTransaction(transaction);
 
 Let's go ahead and practice using lookup tables!
 
-this lab will guide you through the steps of creating, extending, and then using
-a lookup table in a versioned transaction.
+This lab will guide you through creating, extending, and using a lookup table in
+a versioned transaction.
 
 #### 1. Get the starter code
 
-To begin, download the starter code from the starter branch of this
+To begin, download the starter code from the "starter" branch of this
 [repository](https://github.com/Unboxed-Software/solana-versioned-transactions/tree/starter).
-Once you have the starter code, run `npm install` in the terminal to install the
-required dependencies.
+Once you have the starter code, open a terminal and run the following command to
+install the necessary dependencies:
 
-The starter code includes an example of creating a legacy transaction that
-intends to atomically transfer SOL to 22 recipients. The transaction contains 22
-instructions where each instruction transfers SOL from the signer to a different
-recipient.
+```bash
+npm install
+```
 
-The purpose of the starter code is to illustrate the limitation on the number of
-addresses that can be included in a legacy transaction. The transaction built in
-the starter code is expected to fail when sent.
+The starter code illustrates a scenario where a legacy transaction is created to
+transfer SOL to 22 recipients in a single atomic transaction. The transaction
+includes 22 separate instructions, each transferring SOL from the payer (signer)
+to a different recipient.
 
-The following starter code can be found in the `index.ts` file.
+This example highlights a key limitation of legacy transactions when trying to
+accommodate many account addresses within a single transaction. As expected,
+when attempting to send this transaction, it will likely fail due to exceeding
+the transaction size limits.
+
+The key logic can be found in the `index.ts` file of the starter code.
 
 ```typescript
-import { initializeKeypair } from "./initializeKeypair";
 import * as web3 from "@solana/web3.js";
```

Comment:
Just import the things from web3 you will actually use, this helps tree-shaking and will fit in with more of the other  code.